pkg/fab/channel/transactor.go

```diff
@@ -86,11 +86,8 @@ func orderersFromChannelCfg(ctx context.Client, cfg fab.ChannelCfg) ([]fab.Order
 		}
 		//create orderer using channel config block orderer address
```

Comment:
remove this comment?

Reply:
the orderer address is set using the channel config value, so the comment is still valid

Reply:
I will probably move it under the `if !ok` block, it makes more sense there.

Reply:
I ended up not changed the existing behaviour  of this condition, so the comment stays